pkgs/applications/networking/cluster/ktop/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ktop";
+  version = "0.3.5";
+  excludedPackages = [".ci"];
+
+  src = fetchFromGitHub {
+    owner = "vladimirvivien";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-nkIRVt2kqsE9QBYvvHmupohnzH+OBcwWwV16rMePw4I=";
+  };
+
+  vendorHash = "sha256-IiAMmHOq69WMT2B1q9ZV2fGDnLr7AbRm1P7ACSde2FE=";
+  ldflags = [ "-s" "-w" ];
```

Upstream sets the version using ldflags. (See https://github.com/vladimirvivien/ktop/blob/cb3a80f1ea70c6fbc1207c2b2340314aa19962b8/.goreleaser.yml#L34)

We should probably do the same here.

Upstream also sets the commit sha, but we don't have immediate access to that here so setting the version should be fine enough.

```suggestion
  ldflags = [ "-s" "-w" "-X github.com/vladimirvivien/ktop/buildinfo.Version=v${version}" ];
```